cylc/flow/task_pool.py

```diff
@@ -799,66 +813,38 @@ def remove(self, itask, reason=""):
             # Event-driven final update of task_states table.
             # TODO: same for datastore (still updated by scheduler loop)
             self.workflow_db_mgr.put_update_task_state(itask)
-            LOG.debug(f"[{itask}] {msg}")
+            LOG.info(f"[{itask}] {msg}")
```

Comment:
The meaning of "removed" might get confused. In the post task-pool world if a task is "removed" it sounds like it's been subtracted from the workflow rather than removed from the pool.

Perhaps we should log "completed" for natural removal (as opposed to `removed (completed)`) or "remove" for manual "cylc remove" uses?

Reply:
Removing a task from the pool now corresponds to releasing(?) a task from the scheduler's n=0 active window, which is conceptual rather than implementation.

It's useful to see this as part of active task life cycle in the logs, not just in the debug mode.

However, yes, it's probably sufficient to say "completed" now for "natural removal".